src/rtc/mosaic_geobursts.py

```diff
@@ -325,9 +532,9 @@ def compute_weighted_mosaic_raster(list_rtc_images, list_nlooks, geo_filename,
 
 
 
-def compute_weighted_mosaic_raster_single_band(
-        list_rtc_images, list_nlooks, output_file_list, geogrid_in=None,
-        verbose=True):
+def mosaic_multiple_output_files(
+        list_rtc_images, list_nlooks, output_file_list, mosaic_mode,
+        scratch_dir='', geogrid_in=None, temp_files_list=None, verbose=True):
     '''
     Mosaic the snapped S1 geobursts
     paremeters:
```

Comment:
```suggestion
    parameters:
```
- Also please remove `-----------` in the following line.
- `list_rtc` in the following docstring has to be `list_rtc_images`

Reply:
Sorry that I confused to myself. It's a nit-pick but please remove `:` at the end of `parameters`.